src/ray/stats/stats_test.cc

```diff
@@ -88,6 +87,36 @@ TEST_F(StatsTest, F) {
   }
 }
 
+TEST_F(StatsTest, InitializationTest) {
+  // Do initialization multiple times and make sure only the first initialization
+  // was applied.
+  ASSERT_TRUE(ray::stats::StatsConfig::instance().IsInitialized());
+  auto test_tag_value_that_shouldnt_be_applied = "TEST";
+  for (size_t i = 0; i < 20; ++i) {
+    std::shared_ptr<stats::MetricExporterClient> exporter(
+        new stats::StdoutExporterClient());
+    ray::stats::Init({{stats::LanguageKey, test_tag_value_that_shouldnt_be_applied}},
+                     10054, exporter);
```

Comment:
Why the port is 10054?

Reply:
Just a random number haha. I will put it inside a constant variable.